@xen-orchestra/web-core/lib/components/tree-view/TreeLine.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="tree-line">
+    <div
+      class="tree-line-vertical"
+      :class="{ 'tree-line-half-height': halfHeight, 'tree-line-full-height': fullHeight }"
+    />
+    <div class="tree-line-horizontal" :class="{ right }" />
+  </div>
+</template>
+
+<script lang="ts" setup>
+defineProps<{
+  halfHeight?: boolean
+  fullHeight?: boolean
```

Comment:
Is that necessary to have 2 props for that?

Doesn't a TreeLine have to be either `fullHeight` or `halfHeight`?